### DISCUSSION TITLE
support for office365mail

### DISCUSSION THREAD

**Discussion by cityofpae** [directus/directus#16463] [Feature Requests]
Would be great to have support for office365 mail. 

```
EMAIL_TRANSPORT=office365mail
EMAIL_OFFICE365MAIL_CLIENT_ID=
EMAIL_OFFICE365MAIL_TENANT=
EMAIL_OFFICE365MAIL_CLIENT_SECRET=
```

**Comment by rijkvanzanten** [comment]
Doesn't office 365 already support SMTP?

**Comment by cityofpae** [comment]
That might be enabled for personal/home accounts, but we are on a work/organisation account and cannot authenticate via SMTP.
I am using Laravel/Statamic for our Intranet site and authenticating successfully via ClientID,Tenant,Secret... would be great to be able to do the same in Directus.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
